service/modules/synthetic-data-module/synthetic_data.py

```diff
@@ -16,7 +16,7 @@
 from sdgym.constants import CONTINUOUS
 from sdgym.synthesizers.utils import Transformer
 
-from mwem.mwem import MWEMSynthesizer
+from opendp.whitenoise.synthesizers.mwem.mwem import MWEMSynthesizer
```

Comment:
Can this be from opendp.whitenoise.synthesizers.mwem import MWEMSynthesizer.

Also, in 
synthesizers/__init__.py, you can add a shorter import like we do in sql/__init__.py